ui/src/common/storage.js

```diff
@@ -1,9 +1,25 @@
+import localforage from 'localforage';
+
+localforage.config({
+  name: 'inspirehep',
+  driver: [localforage.INDEXEDDB, localforage.LOCALSTORAGE],
+  version: 1.0,
+});
+
+// to not lose user settings saved in localStorage rewrite them to localforage
+for (let i = 0; i < localStorage.length; i += 1) {
+  const key = localStorage.key(i);
+  const value = localStorage.getItem(key);
+
+  localforage.setItem(key, value);
+}
+
```

Comment:
Last thing I promise, could you please add this in a try catch?

Reply:
Done 👾